ghcide/src/Development/IDE/GHC/Util.hs

```diff
@@ -67,12 +67,11 @@ import           Lexer
 import           Module                         (moduleNameSlashes)
 import           OccName                        (parenSymOcc)
 import           Outputable                     (Depth (..), Outputable, SDoc,
-                                                 mkUserStyle, neverQualify, ppr,
-                                                 renderWithStyle,
+                                                 neverQualify, ppr,
                                                  showSDocUnsafe)
-import           PackageConfig                  (PackageConfig)
-import           Packages                       (getPackageConfigMap,
-                                                 lookupPackage')
+-- import           PackageConfig                  (PackageConfig)
+-- import           Packages                       ( -- getPackageConfigMap,
+--                                                  lookupPackage')
```

Comment:
```suggestion
```

Reply:
I think the reason I left them commented out was that they were (incidentally) re-exported from the `.Compat` module and I was a bit unsure if that made it a bit more opaque and if I should import it qualified instead. But yes, I should just remove the commented out code.